powershell_scripts/execute_matlab_tests.ps1

```diff
@@ -0,0 +1,27 @@
+. $PSScriptRoot/powershell_helpers.ps1 <# Imports:
```

Comment:
This script has a number of environment variables it relies on. Either making the variables parameters, or documenting what variables are required, would make running (and debugging) this locally easier